mscr-ui/src/common/components/schema-info/index.tsx

```diff
@@ -266,14 +271,36 @@ export default function SchemaInfo(props: {
 
   return (
     <>
-      <SchemaHeading variant="h2" className={props.raiseHeading ? 'raise-heading' : ''}>
-        {getSchemaData?.metadata.label
-          ? getLanguageVersion({
-              data: getSchemaData.metadata.label,
-              lang,
-            })
-          : t('schema-tree.no-label')}
-      </SchemaHeading>
+      <div className='row d-flex justify-content-between'>
+        <div className='col-6'>
+          <SchemaHeading variant='h2'>
```

Comment:
Good thing the negative margin classname prop is no longer needed. In this case, this was the only place where this heading and that class was used, so would you mind also removing the CSS rules tied to that class in schema-info.styles? Clean out the trash, so to speak.